lib/results.js

```diff
@@ -22,6 +22,7 @@ function Results () {
     this.pass = 0;
     this._stream = through();
     this.tests = [];
+    this._only = 0;
```

Comment:
np - basically you just rebase your local branch as you like - without deleting the branch or pushing any changes to the remote - and then, assuming "origin" is the remote that points to your fork, `git push origin ref-based-only -u -f` - without the "-f" it will refuse to push it.